### DISCUSSION TITLE
lient and empty spaces?

### DISCUSSION THREAD

**Discussion by lunar-studio** [SocolSRT/cloudflare-rules#3] [General]
Nice work. I noticed you had a couple empty spaces at the front.
Also I wasn't sure if you meant 'client' instead of 'lient.'

**Comment by SocolSRT** [comment]
Hello, if you are talking about empty lines here, then this is necessary to block empty user agents, this is a common practice among inexperienced DDosers or during crude flood attacks: (http.user_agent eq "") or (http.user_agent eq " "). 
And if we talk about this "'client' instead of 'lient.'" I didn’t fully understand you, perhaps you asked about cf.client.bot, this is one of the standard cloudflare variables with which you can understand whether the user is a good bot that should be skipped (for example, search engines) or the opposite.

**Reply by lunar-studio** [reply]
I didn’t know if those empty spaces were going to somehow cause issues.
As for ‘lient’ I noticed it in one of the fields after I copied and pasted your expression.
